hooks/useProcessProposal.ts

```diff
@@ -0,0 +1,44 @@
+import Web3 from 'web3';
+import PolyjuiceHttpProvider from '@polyjuice-provider/web3';
+
+import abiLibrary from 'lib/abi';
+
+const providerConfig = {
+  web3Url: 'https://godwoken-testnet-web3-rpc.ckbapp.dev',
+};
+const provider = new PolyjuiceHttpProvider(providerConfig.web3Url, providerConfig);
```

Comment:
Provider and Web3 should be constructed once per whole application.

Reply:
will do this later